.circleci/config.yml

```diff
@@ -90,6 +93,9 @@ jobs:
       - image: memcached
       - image: pafortin/goaws
       - image: circleci/mysql:5.7.27
+        auth:
```

Comment:
It isn't clear to me from their docs if `auth` is needed for each `image` or not. As far as yml structure goes I think it's only attached to one.

Reply:
As above, I'm thinking that any docker hub actions will require auth; I'll ask around a bit on the team to see if I'm misunderstanding